indexer/utils/enrich.py

```diff
@@ -1,269 +0,0 @@
-import itertools
```

Comment:
We don't use this functions anymore?

Reply:
Yes.
The enrich info code from earlier is incorporated into the dataclass initialization.